hs_model_program/static/js/model_program.js

```diff
@@ -0,0 +1,179 @@
+                var term = multiselect[i].getAttribute('parent_metadata');
+
+                // get the selected items that are saved to the resource
+                var selected_terms = data[term];
+
+                // loop through the options in the select list
+                var options = $(multiselect[i]).find('option');
+                for (var j = 0; j < options.length; j++) {
+
+                    // set the selected attribute
+                    if ($.inArray(options[j].value, selected_terms) >= 0) {
+                        $(options[j]).attr("selected", 1);
+                    }
+                }
+            }
+
+            // set the initial value for the date picker
+            var date = new Date(data['date_released']);
+            if (isNaN(date) == false)
+                $('#modelReleaseDate_picker').datepicker('setDate', date);
+
+        },
+        error: function (data) {
+            show_error();
+            console.log('There was an error with model instance GET.')
+        },
+        complete: function (data) {
+            // once the selected fields (html) have been updated, build the jQuery multiselect boxes
+
+            // destroy all multiselect listboxes for the model program resource (i.e.  that match the id mp-multi-select)
+            $('#mp-multi-select.multi-select').multiselect('destroy');
+
+            // get the size of the parent div
+            var width = $('#mp-div-multiselect').css('max-width');
+
+            // rebuild all the multiselect elements for the model program resource
+            $('#mp-multi-select.multi-select').multiselect({
+                buttonWidth: width,
+                maxHeight: 200,
+                numberDisplayed: 1,
+
+                // bind to drop down list close event
+                onDropdownHide: function (event, checked) {
+                    set_metadata_terms(event);
+                }
+            });
+
+            // add pull-right attribute to the dropdown menus
+            var dd = $('.div-multi-select > .btn-group > .multiselect-container.dropdown-menu');
+            for (var i = 0; i < dd.length; i++) {
+                var classname = dd[i].className;
+                if (classname.indexOf(' pull-right ') == -1) {
+                    dd[i].className += ' pull-right';
+                }
+            }
+        }
+    });
+}
+
+function show_error(){
+    $('body > .container').append($alert_error);
+    $(".alert-danger").fadeTo(3000, 500).fadeOut(1000, function(){
+        $(".alert-danger").alert('close');
+    });
+}
+
+/**
+ * reloads the jQuery multi-select widgets on form submission
+ */
+$(document).bind("submit-success", function(event){
+    // reload dynamically generated widgets
+    loadWidgets();
+});
+
+/**
+ * reloads the jQuery multi-select widgets on form submission failure
+ */
+$(document).bind("submit-error", function(event){
+    // reload dynamically generated widgets
+    loadWidgets();
+});
```

Comment:
You can pass a list of events if they have the same handler. Also, `bind()` is replaced with `on()`.

`$(document).on("submit-error submit-success", ...)`